jsx/FilterForm.js

```diff
@@ -132,7 +132,7 @@ class FilterForm extends React.Component {
       } else {         // null and undefined handled here
         filter[key].value = '';
       }
-      filter[key].exactMatch = (type === "SelectElement");
+      filter[key].exactMatch = (type === "SelectElement" || type === "select");
```

Comment:
What's the difference between a "SelectElement" and a "select" type?

Reply:
not sure, i think this was originally developed to detect the React Form components but idk if that is currently functional anywhere.